fpr/migrations/0027_idcommand_tool_disallow_blank.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fpr', '0026_fits_nailgun_compat'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='idcommand',
+            name='tool',
+            field=models.ForeignKey(verbose_name='the related tool', to_field=b'uuid', to='fpr.IDTool', null=True),
```

Comment:
A small thing, but this line I think is 116 characters. It should be brought into line with our recommended coding style before being merged.

Reply:
That code is generated by Django, do you still want me to do it?
Originally, we decided to leave it as is. flake8 was set up to exclude migrations, see:

https://github.com/artefactual/archivematica-fpr-admin/blob/230a99a18fe7557ca1a310d5a1827b0104505952/setup.cfg#L1-L5

Reply:
Thanks for the background @sevein. Merge approval from me as well! :+1: